docs/80_extendingDIRACOS.md

```diff
@@ -0,0 +1,77 @@
+# Extending DIRACOS
+
+Although DIRACOS tries to address most of the VOs needs, it might be that it does not fulfil everything. DIRACOS comes with some extension capabilities described bellow.
+
+## Requesting a package to be added in DIRACOS
+
+If you think a package (RPMs or python) should be added to DIRACOS, please open a [github issue](https://github.com/DIRACGrid/DIRACOS/issues), which will be marked with the `PackageRequest` tag. If it gathers enough thumbs up (Let say 5 from different VO), we will consider adding it.
+
+Note that it is not a guarantee, because we have to take into account other constraints (for example size).
+
+**Spoiler alert**: if you are asking for a new RPM, opening together the issue with a pull request will greatly serve your cause...
+
+
+## Adding pure python packages
+
+
+In case you are extending just pure python packages, you should be able to do it an any machine supported by DIRACOS (SLC6/CC7).
+
+### Initial setup
+
+```
+# Install the few tools needed
+yum install -y jq python-pip git
+
+# Install the diracos machinery (currently from github)
+pip install git+https://github.com/DIRACGrid/DIRACOS.git
+```
+
+### Configuration files
+
+The only thing you need is a json configuration file and a pip `requirements.txt` file containing your new packages.
+
+The configuration file has to contain the following informations:
```

Comment:
I would write here: "The configuration file contain now only the following lines:" as to avoid confusion with, that you have to add these lines to the original diracos.json.